src/main/java/com/powsybl/openloadflow/ac/solver/NewtonRaphson.java

```diff
@@ -84,16 +83,24 @@ public void reportAndLogLargestMismatchByAcEquationType(Reporter reporter, Equat
                         int busPhiRow = equationSystem.getVariable(elementNum, AcVariableType.BUS_PHI).getRow();
                         double busV = equationSystem.getStateVector().get(busVRow);
                         double busPhi = equationSystem.getStateVector().get(busPhiRow);
-                        LOGGER.trace("Mismatch `{}` for {}: {} (element={}) || Bus V /_ PHI = {} /_ {}", acEquationType, equation, equationMismatch, elementId, busV, busPhi);
-                        if (iterationMismatchReporter != null) {
-                            Reports.reportNewtonRaphsonMismatch(iterationMismatchReporter, getEquationTypeDescription(acEquationType), equationMismatch, elementId, busV, busPhi, iteration);
+                        LfBus bus = network.getBus(elementNum);
+                        double busNominalV = bus.getNominalV();
+                        double busSumP = bus.getP().eval() * PerUnit.SB;
+                        double busSumQ = bus.getQ().eval() * PerUnit.SB;
+
+                        LOGGER.trace("Mismatch {} on {}: {}", acEquationType, equation, equationMismatch);
+                        LOGGER.trace("    Bus       Id       : {}", elementId);
+                        LOGGER.trace("    Bus nominalV [  kV]: {}", busNominalV);
+                        LOGGER.trace("    Bus        V [p.u.]: {}", busV);
+                        LOGGER.trace("    Bus      Phi [ rad]: {}", busPhi);
+                        LOGGER.trace("    Bus     sumP [  MW]: {}", busSumP);
```

Comment:
Here I prefer something like "balanceP" or "Pbalance" instead of "sumP". "balanceP" refers directly to the Kirchhoff law.